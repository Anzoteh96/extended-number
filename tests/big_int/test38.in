8865234213580204168737742545475855401238519768498920383417838283098612432549101047777749206587034516650430295466033
338114573700769800166236229670948674002931466865603436145702953763665757388752106958724469606585932488529367220906
