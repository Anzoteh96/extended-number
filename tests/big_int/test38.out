Enter two numbers n and m:
n is 8865234213580204168737742545475855401238519768498920383417838283098612432549101047777749206587034516650430295466033
m is 338114573700769800166236229670948674002931466865603436145702953763665757388752106958724469606585932488529367220906
n + m is 9203348787280973968903978775146804075241451235364523819563541236862278189937853154736473676193620449138959662686939
n - m is 8527119639879434368571506315804906727235588301633316947272135329334946675160348940819024736980448584161900928245127
n * m is 2997464886882149942043061839848486897232212807782460698264123101265592525158279702456103034018479356988754993457530628772539298789234564149520428243710503937633701335452962847290711815408187895362043494477121104580026341330485898
n / m is 26
n % m is 74255297360189364415600574031189877162301629993231043629561485243302740441546266850912996815800271948666747722477
